homework/english/Volha_Zharava/lecture1/task1.md

```diff
@@ -0,0 +1,46 @@
+Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+Is your mum working today?
+Why are this boys talking now?
+Where does he live?
+Does their mother cook very well?
+Lili is not rich, she dosen't drive a Mercedes.
```

Comment:
Doesn't

Reply:
Yes, there is a mistake in line number 6, but there is no more problems. Good job)

Reply:
Thanks, guys)